examples/websocket/nbioclient/client.go

```diff
@@ -13,24 +14,38 @@ import (
 	"github.com/lesismal/nbio/nbhttp/websocket"
 )
 
-func newUpgrader() *websocket.Upgrader {
+var (
+	disableEcho = flag.Bool("disable-echo", true, "diable echo incoming message")
+	verbose     = flag.Bool("verbose", false, "verbose mode")
+	connections = flag.Int("connections", 1000, "number of connecions")
+	msgSize     = flag.Int("msg-size", 1*1024*1024, "message size")
+)
+
+func newUpgrader(connectedChannel *chan *websocket.Conn) *websocket.Upgrader {
 	u := websocket.NewUpgrader()
 	u.OnMessage(func(c *websocket.Conn, messageType websocket.MessageType, data []byte) {
 		// echo
-		time.AfterFunc(time.Second, func() {
-			c.WriteMessage(messageType, data)
-		})
-		log.Println("onEcho:", string(data))
```

Comment:
If use time.AfterFunc, it will execute in another goroutine, then we should not set `ReleaseWebsocketPayload=true`, but we can release the buffer like this:
```golang
time.AfterFunc(time.Second, func() {
	c.WriteMessage(messageType, data)
	mempool.Free(data)
})
```